std/internal/cstring.d

```diff
@@ -52,9 +53,15 @@ body
 }
 
 /**
-Creates temporary $(I C string) with copy of passed text.
+Creates temporary 0-terminated $(I C string) with copy of passed text.
+
+Params:
+    To = character type of returned C string
+    str = string or input range to be converted
+
+Returns:
 
-Returned object is implicitly convertible to $(D const To*) and
+Object is implicitly convertible to $(D const To*) and
```

Comment:
"Object" is easy to confuse with class Object. How about "The type of the returned value".